enterprise/frontend/src/embedding-sdk/types/theme/index.ts

```diff
@@ -79,6 +79,18 @@ export type MetabaseColor = keyof MetabaseColors;
  * in DEFAULT_METABASE_COMPONENT_THEME at [default-component-theme.ts]
  */
 export interface MetabaseComponentTheme {
+  dashboard: {
+    backgroundColor: string;
+
+    card: {
+      backgroundColor: string;
+    };
+  };
+
+  question: {
```

Comment:
If you're curious why `question.backgroundColor` exists when there is already `dashboard.card.backgroundColor`, [see this thread](https://metaboat.slack.com/archives/C063Q3F1HPF/p1717597058789029?thread_ts=1717567761.525549&cid=C063Q3F1HPF).